content/en/docs/setup/install/multicluster/single-network/primary/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Add a Primary Cluster
+description: Describes how to add a primary cluster to your mesh on a flat network.
+weight: 20
+keywords: [multicluster, primary-cluster, deployment]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, you can add
+{{< gloss "primary cluster" >}}primary clusters{{< /gloss >}} to your
+{{< gloss >}}service mesh{{< /gloss >}}.
+
+To give you clean instructions, this section starts right after you complete the
```

Comment:
Probably shouldn't give our opinion as to the quality of our docs :).

Reply:
I'm not commenting on the quality of the docs, just letting readers know what the initial expected state is and why.